code/kobe_choices.py

```diff
@@ -8,8 +8,10 @@
 # Read more about KOBE on https://github.com/exomishra/kobe
 
 # Key differences between KOBE 1.0 and 2.0: 
-# KOBE 1.0 assumes circular orbits, performs approximate calculation of the transit signal, calculates an average number of transits, takes duty_cycle (by default) of 100% and uses a SNR threshold of 7.1 for marking TCEs.
-# KOBE 2.0 accomodates eccentric orbits, makes precise transit signal calculations, improves calculation of number of transits and transit durations, takes the duty_cycle specific to telescope and uses detection efficiency for marking TCEs
+# KOBE 1.0 assumes circular orbits, performs approximate calculation of the transit signal, calculates an average number of transits, 
```

Comment:
we will make it more concise and readable later! (think of lists and references where appropriate)